File: ts-sdk/src/blocks/block.event.ts
import { Block } from "./block";

export class BlockEvent {
  constructor(readonly chainId: number, readonly block: Block) {}

  get network() {
    return this.chainId;
  }

  get blockHash() {
    return this.block.hash;
  }

  get blockNumber() {
    return this.block.number;
  }
}

File: py-sdk/src/forta_bot/blocks/block_event.py
from .block import Block


class BlockEvent:
    def __init__(self, dict):
        self.chain_id: int = dict.get('chain_id', dict.get('network'))
        self.block = Block(dict.get('block', {}))

    @property
    def network(self):
        return self.chain_id

    @property
    def block_hash(self):
        return self.block.hash

    @property
    def block_number(self):
        return self.block.number
